src/patternfly/components/ClipboardCopy/clipboard-copy-group-toggle.hbs

```diff
@@ -0,0 +1,8 @@
+<button class="pf-c-clipboard-copy__group-toggle{{#if clipboard-copy-group-toggle--modifier}} {{clipboard-copy-group-toggle--modifier}}{{/if}}"
+  aria-label="expanded content"
```

Comment:
When the screen reader encounters a `<button>` with `aria-expanded`, it will announce that the element is a button along with the state as "expanded" or "collapsed". For similar cases, like the DataList, we're labelling this button "Details" and then referencing both the button label and primary heading of the datalist item in `aria-labelledby` attribute so that buttons that are duplicated have unique/meaningful labels when announced out of context. For example, the button would be announced like "Node 1 Details button expanded". 

In this case, I'm not exactly sure what the right label would be. The best suggestion I have is "Show Contents", which combined with the input label would be something like "Show Contents URL button collapsed" (assuming "URL" is the input label). Maybe just "Show" would make more sense ("Show URL button collapsed")?

@stacymcauliffe Do you have any wording recommendations for an action that lets you see the full contents of something that is truncated?

Reply:
Implementing the updates I mentioned above would require additional `id` and `aria-labelledby` attributes on the expand button, where aria-labelledby uses the ids of both the expand button and input (or label, as mentioned in my previous comment). The order of ids in this attribute kind of depends on the label we decide to use for button.

Reply:
I changed the text to "Show More" so it reads "Show More Copyable Input", which I think makes sense.

Reply:
@jgiardino - "Show content" is a  good option for truncated content. Google says "Show trimmed content" and I've also seen "Show truncated content"

Reply:
I think "Show More" and "Show Less" work well outside the context of what a screen reader announces, but when the label is combined with the `aria-expanded` attribute, these buttons would be announced like:
- Show More, button collapsed
- Show Less, button expanded

The second one seems really weird to me, like a double-negative, but it could just be me. 🙃 In cases like these, my preference is to rely on `aria-expanded` to communicate state of the button, and to have `aria-label` just describe the contents that are controlled. But this case feels different to me, which is why I think including an action word like "Show" is helpful. And it's also a weird pattern for screen reader users, because the contents of the input won't be truncated when announced by a screen reader. Although maybe there is some benefit for them being able to access the contents in paragraph form outside of the input?

I like the suggestions that @stacymcauliffe shared. I don't have a strong preference for one over the others, but I would pick one of those and just rely on `aria-expanded` to communicate the state of the action.

Your aria attribute updates in the HTML look good!! I just have a few minor refinements to suggest related to all things aria-label/labelledby...

- For the aria-label values, can/should we follow our capitalization standards? E.g.:
`aria-label="copy to clipboard"` >>> `aria-label="Copy to clipboard"`
- I'm not sure how to best communicate when we have strings that should be consistently used across all instances of a component versus strings that should be defined by the consumer. Here are some suggestions, but I'm curious what everyone thinks:
  - `aria-label="[button label text]"` - Since these two cases should be standard, split this one into separate rows for `.pf-c-clipboard-copy__group-toggle` and `.pf-c-clipboard-copy__group-copy` and include the same text we use in the examples instead of `[button label text]`.
  - On the other hand, `aria-label` should be defined by the consumer for `input.pf-c-form-control`. I think this one should be added to the documentation, with something like this?:
    - > Provides an accessible label for the input. Provide a label that describes the contents in the input.

Reply:
Thanks @jgiardino I agree with all of this. 
- I changed the aria-label to "Show Content" but I'm assuming it will still read "Show content, button collapsed" how should I go about hiding the aria-label from the screen reader?
- Capitalized aria labels
- updated the documentation